joomla/joomla-cms
# [A11y] Tables com contact

Pull Request for Issue #.

### Summary of Changes
To be accessible a table must have a header. 
See for examle this checklist: https://dequeuniversity.com/checklists/web/tables

Joomla provides a param show_headings for frontend tables. This removes the table header. 

This PR sets the table header visually-hidden so it is still present for assistive technologies. 

I open some similar PRs to make testing easier. 


### Testing Instructions
You need at least one contact
Make Menu Items for featured contacts and contact category
Set param table_headings in tab "list layouts" to "hide"
See the view and sourcecode ot the tables in frontend 

### Actual result BEFORE applying this Pull Request
There is no table heading visible, also in source code.


### Expected result AFTER applying this Pull Request
There is no table heading visible, but it is in sourcecode with class="visually-hidden"
Make sure that the table heading is shown when table_headings is set to yes, as before.

### Link to documentations
Please select:
